src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,74 @@
  */
 
 public class Plane {
+    private String model;
+    private String color;
+    private int power;
+    private int maxSpeed;
+    private int maxHeight;
 
+    public Plane(String model, String color, int power, int maxSpeed, int maxHeight) {
+        this.model = model;
+        this.color = color;
+        this.power = power;
+        this.maxSpeed = maxSpeed;
+        this.maxHeight = maxHeight;
+    }
+
```

Comment:
to create true Builder pattern, we should have a constructor of Plane with PlaneBuilder as parameter